src/index/updater.rs

```diff
@@ -38,6 +38,7 @@ pub(crate) struct Updater<'index> {
   outputs_cached: u64,
   outputs_inserted_since_flush: u64,
   outputs_traversed: u64,
+  event_sender: Option<tokio::sync::mpsc::Sender<LocationUpdateEvent>>,
```

Comment:
The updater has the index, so we shouldn't need to store the event sender separately.

```suggestion
```

Reply:
Fields should be sorted.